build_defs/kotlin_native/repo.bzl

```diff
@@ -1,13 +1,19 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-_kotlin_native_version = "1.3.50"
+# Release can be found at: https://github.com/JetBrains/kotlin/releases/tag/v<version>
+# For example, this version is located at: https://github.com/JetBrains/kotlin/releases/tag/v1.3.70
+_kotlin_native_version = "1.3.70"
 
 _repo_tmpl = _kotlin_native_version.join([
     "https://github.com/JetBrains/kotlin/releases/download/v",
     "/kotlin-native-{platform}-",
     ".{ext}",
 ])
 
+# `http_archives` expect a sha256 of the artifacts: https://docs.bazel.build/versions/0.17.1/be/workspace.html#http_archive.sha256
+# These can be calculated with https://linux.die.net/man/1/sha256sum on the artifacts from the Bintray repository:
+# https://bintray.com/package/files/jetbrains/kotlin-native-dependencies/dependencies
+# These are typically slow to change.
 _WINDOWS_DEPENDENCIES = [
```

Comment:
I wonder if the 1.3.70 compiler requires newer library deps?

Reply:
As far as I can tell, these don't need updating.